rules/windows/persistence_remote_password_reset.toml

```diff
@@ -14,10 +14,11 @@ passwords to maintain access or evade password duration policies and preserve co
 """
 false_positives = ["Legitimate remote account administration."]
 from = "now-9m"
-index = ["winlogbeat-*", "logs-system.*", "logs-windows.*"]
+index = ["winlogbeat-*", "logs-system.security*", "logs-windows.forwarded*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Account Password Reset Remotely"
+note = "This rule may cause medium to high performance impact due to logic scoping all remote Windows logon activity."
```

Comment:
++

Reply:
I wonder if this is better in guide or setup ...

Thoughts @terrancedejesus @w0rk3r

Reply:
I think it's best where it is...a note about performance.